js/sales.js

```diff
@@ -123,6 +123,65 @@
         return dataCell;
     }
 
+    function getNewStore(event){
+        event.preventDefault();
+        const form = event.target;
+        const storeName = form.querySelector('#store-name').value;
+        const minHourlyCustomers = parseInt(form.querySelector('#min-hourly-customer').value);
+        const maxHourlyCustomers = parseInt(form.querySelector('#max-hourly-customer').value);
+        const avgCookiesPerCustomer = parseInt(form.querySelector('#avg-cookies-per-customer').value);
```

Comment:
`parseInt` will convert decimals into integers, but all of Pat's provided cookie averages per customer have been decimals. We should look into using `parseFloat` here.